From: Cole Robinson <crobinso@redhat.com>
Date: Tue, 2 Apr 2019 15:02:47 -0400
Subject: [PATCH virt-manager] initrdinject: Use full option versions for cpio

Makes it more self documenting IMO

(cherry picked from commit 309840b9535b0e1965ed177b4558652faa256b36)
---
 virtinst/initrdinject.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/virtinst/initrdinject.py b/virtinst/initrdinject.py
index 78c18c28..9a09f20c 100644
--- a/virtinst/initrdinject.py
+++ b/virtinst/initrdinject.py
@@ -30,7 +30,8 @@ def perform_initrd_injections(initrd, injections, scratchdir):
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE,
                                  cwd=tempdir)
-    cpio_proc = subprocess.Popen(['cpio', '-o', '--null', '-Hnewc', '--quiet'],
+    cpio_proc = subprocess.Popen(['cpio', '--create', '--null', '--quiet',
+                                  '--format=newc'],
                                  stdin=find_proc.stdout,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE,
